src/actions/ctl/parse_xml_into_args.cc

```diff
@@ -0,0 +1,63 @@
+/*
+ * ModSecurity, http://www.modsecurity.org/
+ * Copyright (c) 2025 OWASP ModSecurity project
+ *
+ * You may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * If any of the files related to licensing are missing or if you have any
+ * other questions related to licensing please contact Trustwave Holdings, Inc.
```

Comment:
No, I don't think they need to contact Trustwave Holdings Inc., right?

Reply:
Yes, you're definitely right. See 22fee12, it's fixed.